README.md

```diff
@@ -40,7 +40,7 @@ If you want to compile yourself from the sources, you need a working Go
 setup. Then use the provided Makefile (type `make`).
 
 For the most basic setup, just start the binary. To change the address
-to listen on, use the `-web.listen-address` flag. By default, Pushgateway 
+to listen on, use the `-web.listen-address` flag (ie "0.0.0.0:9090"). By default, Pushgateway
```

Comment:
Not sure `9090` is the best port to have in the README - that's most commonly used by Prometheus. How about using the actual default in the example:

```patch
+to listen on, use the `-web.listen-address` flag (e.g. "0.0.0.0:9091" or ":9091"). By default, Pushgateway
```

Reply:
True, gimme a port which is not the standard one ;)